rust_src/src/editfns.rs

```diff
@@ -393,7 +393,7 @@ pub fn char_after(mut pos: LispObject) -> Option<EmacsInt> {
 /// Remaining arguments form a sequence of PROPERTY VALUE pairs for text
 /// properties to add to the result.
 /// usage: (fn STRING &rest PROPERTIES)
```

Comment:
Why are you dropping the min=1? That is accurate.

Reply:
Fixed.